js/imageLoad.js

```diff
@@ -32,7 +32,15 @@ function createLogos(logoArray) {
 
 function insertCategories (logoArray) {
     const map = generateCategoryMap(logoArray);
-    const categories = Object.values(map).sort();
+    const rawcategories = Object.values(map).sort();
+    const categories = []; 
+    rawcategories.forEach(category => {
+        category.replace(" /", "/").replace("/ ","/") //Avoided replace(" / ") so all cases are checked and replaced
```

Comment:
My immediate first impression was, "Woow! This dropdown is long oh! Imagine the experience on a mobile phone" Then I realized it actually something that's inevitable in the end.

WRT this implementation - this is just out of curiosity. Any particular reason why you didn't update the category key to accept an array, instead of separating with a `/`?

We would have stuff like this
```
{
	"title": "ALAT by Wema",
	"filename": "alat_by_wema",
	"category": ["Banking" , "Financial Services"]
}
```

Reply:
Hmmm. This actually makes sense.  
_It's doable_

Reply:
Thanks a lot bro! 💪🏽💪🏽💪🏽

Reply:
Quick One: How do suggest, the category be displayed on the landing page?

Reply:
@dbkonxepts Just to be sure, I get your question, you're asking how I think the categories should be listed in this dropdown down right?
![image](https://user-images.githubusercontent.com/22182946/53807605-48091880-3f50-11e9-8e90-e2d9534895f3.png)

Reply:
Not that. The individual logo category. If we go with array approach.